apps/roam/src/utils/createReifiedBlock.ts

```diff
@@ -21,15 +21,11 @@ const strictQueryForReifiedBlocks = async (
   const query = `[:find ?u ?d
   :in $ ${paramsAsSeq.map(([k]) => "?" + k).join(" ")}
   :where [?s :block/uid ?u] [?s :block/props ?p] [(get ?p :${DISCOURSE_GRAPH_PROP_NAME}) ?d]
-  ${paramsAsSeq.map(([k]) => `[(get ?d :${k}) ?_${k}] [(= ?${k} ?_${k})]`).join(" ")} ]`;
-  // Note: the extra _k binding variable is only needed for the backend query somehow
-  // In a local query, we can directly map to `[(get ?d :${k}) ?${k}]`
-  const result = await Promise.resolve(
-    window.roamAlphaAPI.data.backend.q(
-      query,
-      ...paramsAsSeq.map(([, v]) => v),
-    ) as [string, Record<string, string>][],
-  );
+  ${paramsAsSeq.map(([k]) => `[(get ?d :${k}) ?${k}]`).join(" ")} ]`;
```

Comment:
Note: this line has removed `[(= ?${k} ?_${k})]`.  Because this seems unrelated to the task and there is no comment giving context as to why it was included, it is unclear if this is a mistake.